Backend/Remora.Discord.Gateway/Transport/WebSocketPayloadTransportService.cs

```diff
@@ -276,6 +293,11 @@ public async Task<Result<IPayload>> ReceivePayloadAsync(CancellationToken ct = d
 
             return Result<IPayload>.FromSuccess(payload);
         }
+        catch
+        {
+            // this is most likely reached when the WebSocketState is aborted.
+            return new Results.WebSocketError(WebSocketState.Aborted);
+        }
```

Comment:
See above; furthermore don't make guesses; even if this were the way to go, you can catch `WebSocketException` specifically, and return the current state of the client.

Reply:
Removed this catch.